zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -404,8 +404,8 @@ impl<F: Field> ExecutionConfig<F> {
         &self,
         layouter: &mut impl Layouter<F>,
         block: &Block<F>,
-    ) -> Result<(), Error> {
-        layouter.assign_region(
+    ) -> Result<usize, Error> {
```

Comment:
I don't think you do anything with the returned offset values. Do you need to include this change to the function signatures in this PR?

Reply:
yes you are right . Result<usize, Error> is remain of my first (not successful) version of changes, it is not needed in fact. I will remove it later.